src/darksend.cpp

```diff
@@ -748,14 +748,12 @@ void CDarksendPool::ChargeRandomFees()
 void CDarksendPool::CheckTimeout()
 {
     // check mixing queue objects for timeouts
-    int c = 0;
     std::vector<CDarksendQueue>::iterator it = vecDarksendQueue.begin();
     while(it != vecDarksendQueue.end()) {
         if((*it).IsExpired()) {
-            LogPrint("privatesend", "CDarksendPool::CheckTimeout -- Removing expired queue entry: %d\n", c);
+            LogPrint("privatesend", "CDarksendPool::CheckTimeout -- Removing expired queue (%s)\n", (*it).ToString());
```

Comment:
Isn't it still an entry that's expired, not the whole queue ?  The message doesn't seem right.